asn1/InterledgerProtocol.asn

```diff
@@ -10,12 +10,9 @@ IMPORTS
     Address,
     Timestamp
     FROM InterledgerTypes
-
-    MetadataEntry
-    FROM MetadataEntry
 ;
 
-InterledgerProtocolPayment ::= SEQUENCE {
+InterledgerProtocolPaymentMessage ::= SEQUENCE {
```

Comment:
Okay, just need to make sure whatever we choose we update GenericPacket to use the same.

The other instances are all `InterledgerXRequest` and `InterledgerXResponse` but `InterledgerProtocolPaymentRequest` doesn't seem appropriate...